lib/interceptor.js

```diff
@@ -255,6 +255,12 @@ Interceptor.prototype.match = function match(options, body, hostNameOnly) {
     return false
   }
 
+  const { filteringExternal } = this.scope.scopeOptions || {}
```

Comment:
Coveralls is complaining about a missed branch on this line, and this change may fix it.

```suggestion
  const { filteringExternal } = this.scope.scopeOptions
```